numpy/random/bit_generator.pyx

```diff
@@ -159,8 +159,40 @@ cdef uint32_t mix(uint32_t x, uint32_t y):
     result ^= result >> XSHIFT
     return result
 
+cdef class ISeedSequence():
+    """
+    ISeedSequence() is the abstract base class for SeedSequences.
+
+    See Also
+    --------
+    `SeedSequence`
+    `SeedlessSeedSequence`
+    """
+
+    def __init__(self):
+        raise NotImplementedError('ISeedSequence is a base class and cannot be instantized')
+
+
+    def generate_state(self, n_words, dtype=np.uint32):
+        raise NotImplementedError('seedless SeedSequences cannot generate state')
+
+    def spawn(self, n_children):
+        return [self] * n_children
```

Comment:
These implementations belong in `SeedlessSeedSequence`.

Reply:
ok

Reply:
fixed (thanks @rkern)